urnai/loggers/logger_base.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+
+class LoggerBase(ABC):
+    """
+    Base class for all loggers.
+    """
+
+    @abstractmethod
+    def set_mode(self, train: bool) -> None:
```

Comment:
It took me a while to understand that `train` was about `training` and not 🚋  .

```suggestion
    def set_mode(self, training: bool = True) -> None:
```